setup.cfg

```diff
@@ -7,26 +7,24 @@ python-tag = py3
 # D102: Missing docstring in public method
 # D103: Missing docstring in public function
 # D105: Missing docstring in magic method
-# E501: Line too long
 # W503: line break before binary operator (superseded by W504 line break after binary operator)
 # N806: Variable in function should be lowercase
-ignore = D100,D101,D102,D103,D105,E501,W503,N806
+# E203: Whitespace before ':'
+ignore = D100,D101,D102,D103,D105,W503,N806,E203
```

Comment:
I see you’ve added 266 `noqa: E501` comments, for cases where flake8’s line length check is stricter than black’s. If we have black automatically splitting lines and enforcing line length wherever possible, could we keep E501 disabled with flake8, and trust black’s lenient enforcement will be enough?

I could see a few different patterns where line length goes over 88:

- URLs in comments – there’s little value in enforcing a line length there
- Long import statements that can’t be split further – same here, since we’ll have black and isort splitting wherever possible, it won’t be that useful for flake8 to enforce anything for the few statements that can’t be split (`from wagtail.contrib.search_promotions.templatetags.wagtailsearchpromotions_tags import (`).
- Long identifiers (`def test_adapt_multiple_choice_block_with_existing_blank_choice_and_with_callable_choices(`). 

The majority of cases seem to be with strings – I count 222 lines longer than 88 characters for strings in _test cases_, where to me there is very little value in (automatically) enforcing a line length. There are 17 lines with long strings in non-test code, but here as well the majority look better to me _without_ a split: URLs as strings, URL patterns, help text that’s _just_ over the limit.